gulp-tasks/test.js

```diff
@@ -0,0 +1,40 @@
+const gulp = require('gulp');
+const mocha = require('gulp-spawn-mocha');
+const path = require('path');
+
+const constants = require('./utils/constants');
+const packageRunnner = require('./utils/package-runner');
+const logHelper = require('./utils/log-helper');
+const pkgPathToName = require('./utils/pkg-path-to-name');
+
+const runBundledTests = (packagePath, env) => {
+  logHelper.log(`Running Test Bundles for ` +
+    `${logHelper.highlight(pkgPathToName(packagePath))}.`);
+
+  const bundleTestDirectory = path.join(
+    packagePath, 'test', constants.BUNDLE_BUILD_DIRNAME);
+  return gulp.src(path.join(bundleTestDirectory, env, '*.js'), {read: false})
+  .pipe(mocha());
+};
+
+gulp.task('test:node', gulp.series(
+  packageRunnner(runBundledTests, 'node'),
+));
+
+gulp.task('test:browser', () => {
+  // TODO: This needs implementing
+  return Promise.resolve();
+});
+
+gulp.task('test:sw', () => {
+  // TODO: This needs implementing
+  return Promise.resolve();
+});
+
+gulp.task('test', gulp.series(
+  'test:build-bundles',
```

Comment:
Just to confirm what we discussed via chat, a fresh clone + `npm install` has things working for me.